public/static/docs/command-reference/repro.md

```diff
@@ -45,6 +45,40 @@ files, intermediate or final results. It saves all the data files, intermediate
 or final results into the <abbr>DVC cache</abbr> (unless `--no-commit` option is
 specified), and updates stage files with the new checksum information.
 
+### Parallel stage execution
+
+Currently, `dvc repro` is not able to parallelize execution by itself (see
```

Comment:
How about

```suggestion
Currently, `dvc repro` is not able to parallelize stage execution automatically.
```

and we don't include the link to some `dvc` repo issue if it's not super necessary? It's hard to maintain these links to issues, we tend to forget they're there, and don't remove them when the issues are closed.